### ISSUE TITLE
Select type of connection (2G/3G/4G)

### ISSUE THREAD

**Issue by tobykurien** [tobykurien/BatteryFu#12]
From Sebadamus:

> > It would be great if you add a way to manage the type of connection...  like 2g 3g 4g...

**Comment by dittaeva** [comment]
Indeed, and/or no mobile data connection when WLAN available, and/or no WLAN when no WLAN recognized (might be overly complicated though).

**Comment by Siltaar** [comment]
I'm currently using Network from F-Droid (de.mangelow.network) to easily switch between GSM-only and LTE (preferred) connexions. So indeed I would love BatteryFu to handle it for me.
For instance it could keep the phone in GSM-only (2G) and switch to LTE automatically if all the available data bandwith is currently consumed during a sync time.
